runbot/runbot.py

```diff
@@ -1263,7 +1263,7 @@ def branch_info(branch):
             })
 
         # consider host gone if no build in last 100
-        build_threshold = max(build_ids) - 100
+        build_threshold = max(build_ids or [0]) - 100
```

Comment:
This fix does not work in all situations. If you start with an empty runbot (no repositories in the database), then the build_ids variable is undefined and causes an error in this line.